docs/guides/user-guide/available-environments.md

```diff
@@ -15,13 +15,15 @@ Depending on the environment, StackBlitz IDE includes slightly different feature
 
 | Feature | EngineBlock | WebContainers |
 | --- | --- | --- |
-| Native Node.js environment | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
 | Supported frameworks | <SupportIcon value="star-half" label="" /> Front-end | <SupportIcon value="star" label="" /> Front-end & back-end |
-| Editor | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="yes" label="Available" /> |
-| Preview | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="yes" label="Available" /> |
-| Shareable Preview URL | <YepNope value="yes" label="Supported" /> | <SupportIcon value="no" label="Not supported" /> |
-| Console | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="no" label="Not available" /> |
-| Terminal | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
+| Supported package managers | <SupportIcon value="star-half" label="" /> Turbo v1 | <SupportIcon value="star" label="" /> Turbo v2, pnpm, yarn v1 |
+| Full Node.js environment | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
+| Classic editor | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="yes" label="Available" /> |
+| [Codeflow](/codeflow/working-in-codeflow-ide) (beta) | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
+| [Web Publisher](/codeflow/content-updates-with-web-publisher) (beta) | <SupportIcon value="no" label="Not available" /> | <SupportIcon value="yes" label="Available" /> |
+| Shareable preview URL | <SupportIcon value="yes" label="Available" /> | <SupportIcon value="no" label="Not available" /> |
```

Comment:
I removed the "Preview" line because it's available in both environments, most users of StackBlitz will have seen a preview running at some point before ending up on this documentation page, and also we have previews in all 3 editors: classic editor, Web Publisher and Codeflow IDE so it wasn't clear what "Preview" would refer to.

Reply:
Thanks for this context. Totally makes sense. I think that for most folks "preview" is an integral part of StackBlitz and I feel like it's also self-explanatory.